lib/routes/nm/addAndActivateConnection.ts

```diff
@@ -0,0 +1,115 @@
+import * as Promise from 'bluebird';
```

Comment:
Please import Bluebird as `Bluebird` to avoid clashing with the already existing `Promise`